### DISCUSSION TITLE
How to crop and mesh from the command line

### DISCUSSION THREAD

**Discussion by mkim-MASI** [NVlabs/instant-ngp#1371] [Q&A]
In the GUI, I can crop the scene to be more centered on the object rather than including the entire background, and then mesh the cropped portion. Trying to create a mesh from the command line with run.py gives me the entire background and my object does not show up in the final mesh. Is there a way I can change the crop size / crop box from the command line and mesh that instead of the entire scene?

**Comment by JuanMaParraU** [comment]
Any ideas how to crop the mesh before saving it from the command line?

**Comment by Timothy-pop** [comment]
I also have the same issue. How can I set a 3D rectangular bounding box for the area of interest from the command line, and set the corresponding parameters to export the Mesh
